features/attached_enable.feature

```diff
@@ -114,6 +114,39 @@ Feature: Enable command behaviour when attached to an UA subscription
            | trusty  | libgit2-0 | https://esm.ubuntu.com/ubuntu/      |
            | xenial  | libkrad0  | https://esm.ubuntu.com/infra/ubuntu |
 
+    @wip
+    @series.xenial
+    @series.bionic
+    @series.focal
+    Scenario Outline: Attached enable of a know service shows update in a ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua enable esm-infra` with sudo
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            ESM Infra is already enabled.
+            See: sudo ua status
+            """
+        When I run `apt install -y <pkg-version>` with sudo
+        And I run `apt update` with sudo
+        Then stdout matches regexp
+        """
+        \d* of the updates \w* from UA Infrastructure ESM
```

Comment:
this is a more permissive regex than mine (is|are). I think it's probably better to be strict if we can (also because it's potentially easier to read what that expected message should be.